graders/python/requirements.txt

```diff
@@ -1,28 +1,28 @@
 beautifulsoup4==4.12.3
-bokeh==3.4.1
+bokeh==3.4.2
 colormath==3.0.0
 defusedxml==0.7.1
-Faker==25.3.0
-folium==0.16.0
-ipython==8.25.0
+Faker==26.0.0
+folium==0.17.0
```

Comment:
The only breaking change here was the removal of Python 3.7 support, which doesn't affect us: https://github.com/python-visualization/folium/releases/tag/v0.17.0